docs/tutorials/intro.md

```diff
@@ -74,21 +78,22 @@ fn main() {
     println!("You are using: {}", loc);
 }
 ```
-*Notice:* `ICU4X` canonicalized the locales's syntax which uses lowercase letter for the language portion.
+*Notice:* Here, `ICU4X` canonicalized the locales's syntax which uses lowercase letters for the language portion.
```

Comment:
Referring to the above^

"¡Hola amigo!" is a really bad string to use, as it does not take the user's gender into account. We're trying to do good i18n.